pom.xml

```diff
@@ -72,21 +72,88 @@
 
   <dependencies>
     <!-- Dropwizard -->
+    <!-- The exclusions are required as Dropwizard seems to be having issues regarding the versioning
+     of packages its internal components use -->
     <dependency>
       <groupId>io.dropwizard</groupId>
       <artifactId>dropwizard-core</artifactId>
       <version>${dropwizard.version}</version>
+      <exclusions>
+        <exclusion>
```

Comment:
For these - where were the conflicts originating? Can you post the conflict output?

Reply:
Just posted a screencap as a new comment

Reply:
I figured out how to simplify this a bit, if we pull in the dropwizard-bom:
```
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>io.dropwizard</groupId>
        <artifactId>dropwizard-bom</artifactId>
        <version>${dropwizard.version}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
```
And remove all specified versions, and exclusions, Dropwizard has already set everything up for us to not get conflicts.